config/externalname.go

```diff
@@ -42,6 +42,11 @@ var TerraformPluginFrameworkExternalNameConfigs = map[string]config.ExternalName
 	// AppConfig Environments can be imported by using the environment ID and application ID separated by a colon (:)
 	// terraform-plugin-framework
 	"aws_appconfig_environment": appConfigEnvironment(),
+
+	// eks
+	//
+	// Imported using the cluster_name and association_id separated by a comma (,)
```

Comment:
This import syntax does not apply to the framework architecture, to import consumers should set `association_id` as the external name and pass `cluster_name` in `spec.forProvider`

Reply:
Thanks for the catch, done.